backend/vendor/github.com/go-pkgz/auth/auth.go

```diff
@@ -265,15 +265,14 @@ func (s *Service) AddDirectProvider(name string, credChecker provider.CredChecke
 }
 
 // AddVerifProvider adds provider user's verification sent by sender
-func (s *Service) AddVerifProvider(name, msgTmpl string, sender provider.Sender) {
+func (s *Service) AddVerifProvider(name, sender provider.Sender) {
```

Comment:
vendor should not be modified like this. The only way we get dependency updates is "go mod ... " commands

Reply:
hm, I think I should check what I'm doing after "go to definition". My mistake, sorry